packages/design-system/src/breadcrumbs.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { Breadcrumbs as MuiBreadcrumbs } from "@mui/material";
+import cn from "classnames";
+
+interface BreadcrumbItem {
+  label: string;
+  href: string;
+}
+
+interface BreadcrumbsProps {
+  items: BreadcrumbItem[];
+  className?: string;
+}
+
+const Breadcrumbs = React.forwardRef<HTMLDivElement, BreadcrumbsProps>(
+  ({ items, className }, ref) => (
+    <MuiBreadcrumbs
```

Comment:
Ja jsem dal trochu premysleni nejake trategii pridavani knihoven a dosel jsem k tomu, ze bych se rad vyhnul MUI. Muzes to, prosim, udelat bez ni.